test/testinput/bufr_ncep_adpsfc.yaml

```diff
@@ -0,0 +1,526 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+# #
+# # This software is licensed under the terms of the Apache Licence Version 2.0
+# # which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "testinput/gdas.t12z.adpsfc_thinned.tm00.bufr_d"
+
+      exports:
+        variables:
+          timestamp:
+            datetime:
+              year: "*/YEAR[1]"
+              month: "*/MNTH[1]"
+              day: "*/DAYS[1]"
+              hour: "*/HOUR[1]"
+              minute: "*/MINU[1]"
+          durationOfTimeRelatingToDepthOfFreshSnow:
+            query: "*/.DTHDOFS"
+          durationOfTimeRelatingToMinimumTemperature:
+            query: "*/.DTHMITM"
+          durationOfTimeRelatingToMaximumTemperature:
+            query: "*/.DTHMXTM"
+          durationOfTimeRelatingToTotalPrecipitation:
+            query: "*/.DTHTOPC"
+          relationshipToHorizontalVisibility:
+            query: "*/.REHOVI"
+          PressureChangePast24Hour:
+            query: "*/24PC"
+          PressureChangePast3Hour:
+            query: "*/3HPC"
+          altimeterSettingQnh:
+            query: "*/ALSE"
+          characteristicOfPressureTendency:
+            query: "*/CHPT"
+          cloudAmount:
+            query: "*/CLAM"
+          latitude:
+            query: "[*/CLAT, */CLATH]"
```

Comment:
I thought that you'd usually want CLATH first (might not matter) as that it is prefered over CLAT (high res vs low resolution).

Reply:
Normally yes, but in ADPSFC, only one of CLAT or CLATH will exist in each report. For example, only CLAT exists in subtypes NC000000, 000001, 000002 and 000007, while CLATH exists in 000100, 000101, and 000102. Same with CLON/CLONH. So both are required